enterprise/api/v1/views/enterprise_group.py

```diff
@@ -158,6 +162,8 @@ def assign_learners(self, request, group_uuid):
         except models.EnterpriseGroup.DoesNotExist as exc:
             raise Http404 from exc
         if requested_emails := request.POST.dict().get('learner_emails'):
+            budget_expiration = request.POST.dict().get('budget_expiration')
+            catalog_uuid = request.POST.dict().get('catalog_uuid')
```

Comment:
fyi, you can use `request.data.get('whatever payload key)` here: https://www.django-rest-framework.org/api-guide/requests/#data